lab_5_scrapper/scrapper.py

```diff
@@ -90,25 +154,39 @@ def __init__(self, config: Config) -> None:
         """
         Initializes an instance of the Crawler class
         """
-        pass
+        self.config = config
+        self.urls = []
+        self._seed_urls = self.config.get_seed_urls()
 
     def _extract_url(self, article_bs: BeautifulSoup) -> str:
         """
         Finds and retrieves URL from HTML
         """
-        pass
+        return article_bs.get('href')
 
     def find_articles(self) -> None:
         """
         Finds articles
         """
-        pass
+        art_num = 0
+        for url in self._seed_urls:
+            response = make_request(url, self.config)
+            if response.status_code != 200:
+                continue
+            main_bs = BeautifulSoup(response.text, 'lxml')
+            feed_lines = main_bs.find_all('a', {'class': 'd-block mb-0'})
+            for line in feed_lines:
+                self.urls.append(self._extract_url(line))
+                art_num += 1
```

Comment:
you always know number of collected urls as they are stored in collection. therefore, no need in additional counter

Reply:
Done